README.txt

```diff
@@ -49,6 +49,7 @@ Pre Installation:
    options mlx4_core fast_drop=1
    options mlx4_core log_num_mgm_entry_size=-1
    And restart the driver: /etc/init.d/openibd restart
+   - or reboot while openibd in not exist.
```

Comment:
I would like to avoid mentioning about `openibd` service as far it might be replaced with `rdma` service in some systems as RedHat, Fedora, SentOS.
Or use something like
```
Restart openib or rdma service depending on your system configuration
```